### PR TITLE
Fix Bootstrap checkbox display issue

### PR DESCRIPTION
**Pull request by jtheisen**
I didn't manage to rebase properly, so I created a new pull request.

This is another attempt at fixing #103.

I put the new option in and the name and the defaults are as you suggested.

### CODE REVIEW

File: ChameleonForms/Component/Config/FieldConfiguration.cs [hunk 1]
```diff
@@ -697,6 +704,16 @@ public IFieldConfiguration WithoutInlineLabel()
         }
 
         /// <inheritdoc />
+        public IFieldConfiguration InlineLabelWrapsElement(bool wrapElement = true)
+        {
+            this.ShouldInlineLabelWrapsElement = true;
```
Comment (robdmoore) [inline ChameleonForms/Component/Config/FieldConfiguration.cs]: Minor: we don't use `this` in the rest of the code

File: ChameleonForms/Component/Config/FieldConfiguration.cs [hunk 2]
```diff
@@ -697,6 +704,16 @@ public IFieldConfiguration WithoutInlineLabel()
...
+        {
+            this.ShouldInlineLabelWrapsElement = true;
+            return this;
+        }
+
+        /// <inheritdoc />
         public bool HasInlineLabel { get; private set; }
+
+        /// <inheritdoc />
+        public bool ShouldInlineLabelWrapsElement { get; private set; }
```
Comment (robdmoore) [inline ChameleonForms/Component/Config/FieldConfiguration.cs]: From a grammatic point of view this should be `ShouldInlineLabelWrapElement`

**Comment by robdmoore** [comment]
Looking good. 2 more things I think are needed.

Change the version number of the bootstrap dependency in `ChameleonForms.TwitterBootstrap3.nuspec` to match the version you've updated to.

Add in a test for each of the two instances where there setting the new field configuration changes the HTML.

**Comment by jtheisen** [comment]
That should be all you mentioned.

I'm not sure if the tighter bootstrap version requirement is necessary: It's not that ChameleonForms.Twitterbootstrap requires the new version. The new version just requires the fix.

**Comment by jtheisen** [comment]
Can I get an update on this?

**Comment by robdmoore** [comment]
Sorry for the delay.

Thanks for the contribution!

**Comment by jtheisen** [comment]
No problem - do you have any rough idea when there will be a release, and especially when the NuGet package will be updated?

**Comment by robdmoore** [comment]
Hi @jtheisen.

It will be released once #132 is merged and we publish v2.0.

Sorry for the delay. We wouldn't normally wait so long to release, but there were a few breaking changes we wanted to bunch up.

cc @MattDavies

**Comment by jtheisen** [comment]
No problem, thank you for your efforts.

**Pull request merged**